virt_lightning/shell.py

```diff
@@ -228,6 +228,14 @@ def main():
         default=True,
     )
     start_parser.add_argument("distro", help="Name of the distro", type=str)
+    start_parser.add_argument(
+        "--disks",
```

Comment:
Call it just `--disk` please. I would prefer we accept several times the same optional argument like this: `vl start fedora --disk 10 --disk 32`.